staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -17,35 +17,39 @@ limitations under the License.
 package features
 
 import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/featuregate"
 )
 
 const (
 	// Every feature gate should add method here following this template:
 	//
-	// // owner: @username
-	// // alpha: v1.4
+	// owner: @username
+	// alpha: v1.4
 	// MyFeature featuregate.Feature = "MyFeature"
 	//
 	// Feature gates should be listed in alphabetical, case-sensitive
 	// (upper before any lower case character) order. This reduces the risk
 	// of code conflicts because changes are more likely to be scattered
 	// across the file.
 
-	// owner: @khenidak
-	// alpha: v1.15
+	// owner: @alexanderConstantinescu
+	// kep: http://kep.k8s.io/3458
+	// beta: v1.27
```

Comment:
Sorry if I missed this in the KEP, but what's the reason for introducing this feature as Beta and enabled by default?

Reply:
The genesis to this lies here: https://github.com/kubernetes/kubernetes/pull/115204#issuecomment-1399095942

Reply:
Ah I see the comments earlier in the PR now discussing this.